babel.config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
@jlipps

`babel.config.json` (or `babel.config.js`) is the "root" babel config as of Babel v7.  a `.babelrc` file in any package will merge or override (not clear...) the two config files.  

this is roughly the same as the `.babelrc` file in `gulp-plugins` -- but trying to output some source maps here for debugging & stack traces.  I'm _pretty sure_ this file is not yet used anywhere _except_ when debugging in an IDE.